IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowse.LeftPanelTests.Remotes.cs

```diff
@@ -20,25 +20,34 @@ public class FormBrowse_LeftPanel_RemotesTests
         // Created once for the fixture
         private ReferenceRepository _referenceRepository;
 
+        // Track the original setting value
+        private bool _originalShowAuthorAvatarColumn;
+        private bool _showAvailableDiffTools;
+
         // Created once for each test
         private GitUICommands _commands;
         private IConfigFileRemoteSettingsManager _remotesManager;
-        private bool _originalShowAuthorAvatarColumn;
         private static readonly string[] RemoteNames = new[] { "remote1", "remote5", "remote3", "remote4", "remote2" };
 
         [OneTimeSetUp]
         public void SetUpFixture()
         {
+            // Remeber the current settings...
```

Comment:
nit pick, some other occurrences too
```suggestion
            // Remember the current settings...
```

Reply:
Thank you... tired eyes